benchmarks/TF/CMakeLists.txt

```diff
@@ -30,6 +30,20 @@ set(MINILM_L12_H384_UNCASED_INT32_SEQLEN128_MODULE
     "1x128xi32,1x128xi32,1x128xi32"
 )
 
+set(VIT_B16_CLASSIFICATION_FP32_MODULE
+  NAME
+    "VitB16Classification"
+  TAGS
+    "fp32"
+  SOURCE
+    # Mirror of https://tfhub.dev/sayakpaul/vit_b16_classification/1.
+    "https://storage.googleapis.com/iree-model-artifacts/vit_b16_classification_1.tar.gz"
```

Comment:
The file name needs to end with `-tf-model.tar.gz`; otherwise it won't be considered as a TF model.

Reply:
Thanks!